### PR TITLE
Add resources for PR testing.

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

File: resources/jenkins/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+These scripts are intended to be executed within a Jenkins build
```
Comment (bgrozev) [inline resources/jenkins/README.md]: I think these are superfluous. I can't of anything other than `./build.sh` we'd want to add to the jenkins-specific script.

Comment (bgrozev) [reply resources/jenkins/README.md]: @bbaldino we didn't go out of our way to remove them in rtp and jmt, but I think you'll agree we don't need to add them

Comment (JonathanLennox) [reply resources/jenkins/README.md]: I just copied what was in jitsi-rtp -- I don't know what's actually needed.

Comment (bbaldino) [reply resources/jenkins/README.md]: Yeah we can skip them

Comment (JonathanLennox) [reply resources/jenkins/README.md]: So just `resources/build.sh` and not anything in `resources/jenkins`?

Comment (bgrozev) [reply resources/jenkins/README.md]: Yes

Comment (bbaldino) [reply resources/jenkins/README.md]: Yeah just make sure build.sh doesn't reference Jenkins vars

**Pull request merged**